AssetTrackerDevice/DeviceRunner.cs

```diff
@@ -40,11 +40,15 @@ protected override async Task ExecuteAsync(CancellationToken stoppingToken)
                 client.Property_Interval.PropertyValue.Value = p.Value;
                 return await Task.FromResult(ack);
             };
+
             await client.Property_Interval.InitPropertyAsync(client.InitialState, default_interval, stoppingToken);
-            await client.Property_FrameworkVersion.ReportPropertyAsync(Environment.Version.ToString());
-            await client.Property_Manufacturer.ReportPropertyAsync("Contoso");
-            await client.Property_SDKVersion.ReportPropertyAsync(typeof(IoTHubPnPClient).Assembly.GetName().Version.ToString());
 
+            client.Property_FrameworkVersion.PropertyValue = Environment.Version.ToString();
+            client.Property_Manufacturer.PropertyValue = "Contoso";
+            client.Property_SDKVersion.PropertyValue = (typeof(IoTHubPnPClient).Assembly.GetName().Version.ToString());
+
+            await client.ReportPropertyAsync(client.AllReadOnlyProperties, stoppingToken);
```

Comment:
@wickste  would this work?

we might need a same approach to send multiple telemetries in one message